next.html

```diff
@@ -0,0 +1,625 @@
+                                  <div tabindex="0">
+                                    <p>LeaderImpact NEXT is a gathering of early-career leaders who inspire each other to grow personally, professionally, and spiritually, in order to impact the world. It starts now and it starts with you. Who's next?
+                                        <br><br>
+                                        Are you a young professional looking to engage your personal, professional and spiritual life? LeaderImpact NEXT seeks to give our young leaders a place to engage, equip and learn from a variety of resources and tools.</p>
+                                  </div>
+                                </div>
+                            </div>
+                            <div class="button">
+                                <a id="button-a1f53d50c2" class="cmp-button" data-cmp-clickable="" data-cmp-data-layer="{&quot;button-a1f53d50c2&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/button&quot;,&quot;dc:title&quot;:&quot;Anchor Button&quot;,&quot;xdm:linkURL&quot;:&quot;/content/core-components-examples/library/core-content/button.html&quot;}}" href="/content/core-components-examples/library/core-content/button.html">
+                                   <span class="cmp-button__text">Find a Next Group</span>
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+          </div>
+          <!-- STEP BOXES -->
+          <div class="container responsivegrid step-boxes aem-GridColumn aem-GridColumn--default--12">
+            <div id="container-6eef6707d1" class="cmp-container">
+              <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                <div class="title cmp-examples-title--background-color aem-GridColumn aem-GridColumn--default--12">
+                  <div data-cmp-data-layer="{&quot;title-7c16f13db5&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/title&quot;,&quot;dc:title&quot;:&quot;Lorem Ipsum&quot;}}" id="title-7c16f13db5" class="cmp-title">
+                    <h2 class="cmp-title__text">Next Steps</h2>
+                  </div>
+                </div>
+                <div class="container responsivegrid content-box aem-GridColumn--default--none aem-GridColumn aem-GridColumn--default--4 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0" style="background-image:url(./assets/step1.png);background-size:cover;background-repeat:no-repeat;background-position:top center;">
+                  <div id="container-a4ac1a4837" class="cmp-container">
+                    <div class="text sub-title aem-GridColumn aem-GridColumn--default--12">
```

Comment:
@Prystal254 I wonder if each of these could utilize a teaser instead - image, pre-title, title, description, and CTA.

Also, is the current effect of removing the yellow gradient on hover something you added? I don't see it implied or mentioned in the designs and I feel like it implies interactivity when there isn't any until you hover over the CTA.

Reply:
@Prystal254 Better, but the teasers need the `container responsivegrid` classes removed from lines 403, 424, and 445.